locale/en/blog/vulnerability/july-2022-security-releases.md

```diff
@@ -45,3 +45,25 @@ Subscribe to the low-volume announcement-only nodejs-sec mailing list at https:/
 The OpenSSL versions 1.1.1q and 3.0.5 will be released July, 5th 2022. The announcement can be found here: https://mta.openssl.org/pipermail/openssl-announce/2022-July/000229.html
 
 The Node.js team will evaluate the OpenSSL changes when they are available and then announce the new target date by Thursday or earlier.
+
+## (Update 05-Jul-2022) OpenSSL Security Release Assessement
+
+The vulnerabilities in the OpenSSL Security releases of Jul 5th 2022 affects Node.js v18.x, v16.x, and v14.x.
+
+* Node.js is affected by **one** MODERATE vulnerability on Windows 32-Bit x86.
+
+The [next](https://nodejs.org/en/blog/vulnerability/july-2022-security-releases/) Node.js Security Release will include
+the updated OpenSSL versions.
```

Comment:
```suggestion
The security release will be delayed so that we can incorporate the the updated OpenSSL versions.  We
will post another update once we have an updated target for the release date.
```